platform/core/src/services/MultiMonitorService.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * This service manages multiple monitors or windows.
+ */
+export class MultiMonitorService {
+  public readonly numberOfScreens: number;
+  private windowsConfig;
+  private screenConfig;
+  private launchWindows = [];
+  private commandsManager;
+
+  public readonly screenNumber: number;
+  public readonly isMultimonitor: boolean;
+
+  public static REGISTRATION = {
+    name: 'multiMonitorService',
+    create: ({ configuration, commandsManager }): MultiMonitorService => {
+      const service = new MultiMonitorService(configuration, commandsManager);
+      return service;
+    },
+  };
+
+  constructor(configuration, commandsManager) {
+    const params = new URLSearchParams(window.location.search);
+    const screenNumber = params.get('screenNumber');
+    const multimonitor = params.get('multimonitor');
```

Comment:
Since the browser doesn't allow us to check whether a window with a specific name exists without calling `window.open` then I think another param, eg `alwaysOpen`, could prove useful. This would call `window.open` on all configured windows on init so it gets all references right away. 

The use case for this is making development easier as every code change triggers a refresh (webpack build) which loses references to previously opened windows. 

For example, if I have 2 windows opened, I refresh, call some multi window command, then it fails because it doesn't know about the other window(s) because it hasn't technically opened them yet.

Reply:
That is a future enhancement - I'd like to proceed with a very simple multi monitor setup to start with, and then see how it works.  Right now, the first window is always window 0 and can't actually position itself correctly due to browser limitations..  Other than that, the other windows will refresh themselves when they are needed, because the system detects the missing references, and clears them if on clears another window.

Reply:
I don't know of a good solution for refreshing either window and still being able to access the other monitor cleanly without sending a new open window command.  The problem is that the references are gone.  For me, if I refresh W0 then sending a command to W1 will work, it just does it by reopening the window by name.  The other way around, W1 will work if I first load a new study into W1 from W0.  That is, refresh W1, load a study from W0 into W1 and then one can run commands.

Reply:
A back end service would work of course, but that requires specific browser support.

Reply:
There is apparently a way to restore the references without causing a secondary reload, but it requires naming the original window correctly as well as not changing the URL of any secondary windows, so I'm going to have to redo a bit of the code to get that working.  Stay tuned.